src/components/LazyLottie.tsx

```diff
@@ -0,0 +1,56 @@
+import { Container } from 'constate'
+import * as React from 'react'
+import { Mount } from 'react-lifecycle-components'
+import ReactLottieComponent, { ReactLottieProps } from 'react-lottie'
+
+interface State {
+  animationData: any | null
+  ReactLottie: (typeof ReactLottieComponent) | null
+}
+
+interface Actions {
+  setLoadedAnimationData: (animationData: any) => void
+  setLoadedLottie: (ReactLottie: typeof ReactLottieComponent) => void
+}
+
+export const LazyLottie: React.SFC<ReactLottieProps> = (props) =>
+  process.env.NODE_ENV === 'test' ? null : (
+    <Container<State, Actions>
+      initialState={{ animationData: null, ReactLottie: null }}
+      actions={{
+        setLoadedAnimationData: (animationData: any) => () => ({
+          animationData,
+        }),
+        setLoadedLottie: (ReactLottie: typeof ReactLottieComponent) => () => ({
+          ReactLottie,
+        }),
+      }}
+    >
+      {({
+        animationData,
+        setLoadedAnimationData,
+        ReactLottie,
+        setLoadedLottie,
+      }: State & Actions) => (
+        <Mount
+          on={() => {
+            props.options.animationData.then(setLoadedAnimationData)
+            import('react-lottie').then((lottieModule) => {
```

Comment:
maybe we could add a chunk name here? like so https://github.com/webpack/webpack/tree/master/examples/code-splitting-specify-chunk-name